lib/linux_admin/time_date.rb

```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class TimeDate
+    extend Common
+    COMMAND = 'timedatectl'
+
+    def self.timezone
+      result = run(cmd(COMMAND), :params => ["status"])
+      result.output.split("\n").each do |l|
+        return l.split(':')[1].strip if l =~ /Time.*zone/
+      end
+    end
+
+    def self.set_system_time(time)
+      run!(cmd(COMMAND), :params => ["set-time", "#{time.strftime("%F %T")}", :adjust_system_clock])
+    end
+
+    def self.set_system_timezone(location, city)
```

Comment:
I kind of like adding system like the `system_time=` method.  I think it communicates that we are altering the underlying system setting rather than just storing data in class variables or something like that.